File: apps/macos/DNDSync/LanConstants.swift
enum LanConstants {
    /// Bonjour service type. Must match `NSBonjourServices` in Info.plist.
    static let serviceType = "_dndsync._tcp"
    static let pairId = "dndsync-dev"
    static let pairTxtKey = "pair"
    static let macInstanceName = "dndsync-dev-mac"
    static let androidInstanceName = "dndsync-dev-android"
    static let senderMac = "mac"
    static let senderAndroid = "android"
    static let protoVersion: UInt32 = 1
    static let echoWindowMs: Int64 = 1000
    static let maxFrameBytes = 64 * 1024
}

File: apps/android/app/src/main/java/com/dndsync/android/lan/LanConstants.kt
package com.dndsync.android.lan

/** Bonjour/NSD names and LAN frame limits. Wire version/senders live in [com.dndsync.android.sync.Wire]. */
object LanConstants {
    const val SERVICE_TYPE = "_dndsync._tcp."
    const val PAIR_ID = "dndsync-dev"
    const val PAIR_TXT_KEY = "pair"
    const val ANDROID_INSTANCE_NAME = "dndsync-dev-android"
    const val MAX_FRAME_BYTES = 64 * 1024
}
